### PR TITLE
 week5 react implemented

### PR DESCRIPTION
**Pull request by daxiousm**

### CODE REVIEW

File: react/week5/githubuser-search/src/App.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+               path="/" exact component={Home}>
+           </Route>
+
+           <Route 
+                path="/about" component={About}>
+           </Route>
+
+           <Route 
+               path="/user/:id" component={User}>
+           </Route>
```
Comment (benna100) [inline react/week5/githubuser-search/src/App.js:35]: Nice!

File: react/week5/githubuser-search/src/Components/SearchBox/SearchBox.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+const SearchBox = ( {searchChange} ) => {
+    return (
+        <div className='pa1 tc'>
+            <form onSubmit={searchChange}>
```
Comment (benna100) [inline react/week5/githubuser-search/src/Components/SearchBox/SearchBox.js:6]:
```suggestion
            <form onSubmit={searchSubmitted}>
```

File: react/week5/githubuser-search/src/Components/User/User.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            src={user.avatar_url} className="br-100 h4 w4 dib ba b--black-05 pa2"
+            alt='user-avatar'  
+            />
+        </div>
+        <div className='tc'>
+          <h1>{user.login}</h1>
+           <ul className="list pl0 measure center">
+             <li className='lh-copy pv3 ba bl-0 bt-0 br-0 b--dotted b--black-30'> <a className="f4 fw6 db black link hover-dark-green" href={user.repos_url}>Repositiories </a> </li>
+             <li className='lh-copy pv3 ba bl-0 bt-0 br-0 b--dotted b--black-30'> <a class="f4 fw6 db black link hover-green" href={user.gists_url}>Gists </a> </li>
+             <li className='lh-copy pv3 ba bl-0 bt-0 br-0 b--dotted b--black-30'> <a class="f4 fw6 db black hover-light-purple" href={user.followers_url}>Followers </a> </li>
```
Comment (benna100) [inline react/week5/githubuser-search/src/Components/User/User.js:31]: What is this crazy class naming? Im sure its some css framework. 

Super nice here though!

File: react/week5/githubuser-search/src/Components/User/index.js [hunk 4]
```diff
@@ -0,0 +1,2 @@
+import User from './User'
+export default User
```
Comment (benna100) [inline react/week5/githubuser-search/src/Components/User/index.js:2]: A bit overkill in my opinion

File: react/week5/githubuser-search/src/Components/Users/Users.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+import { Link } from 'react-router-dom'
+
+const Users = () => {
+
+    const[input, setInput] = useState('')
+    const[isLoading,setIsLoading]=useState(false)
+    const[error,setError]=useState(false)
+    const[users, setUsers] = useState([])
+    const url = `https://api.github.com/search/users?q=${input}`
```
Comment (benna100) [inline react/week5/githubuser-search/src/Components/Users/Users.js:13]:
```suggestion
    const usersSearchUrl = `https://api.github.com/search/users?q=${input}`
```

File: react/week5/githubuser-search/src/Components/Users/Users.js [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+        setError(true)
+        setIsLoading(false)
+        console.error(err)
+    })
+},[url,input])
+
+
+const submitHandler = e => {
+    e.preventDefault()
+    setInput(e.target.elements.query.value)
```
Comment (benna100) [inline react/week5/githubuser-search/src/Components/Users/Users.js:39]: Super nice! Your indentation does not seem to work here

**Review by benna100 — CHANGES_REQUESTED** [review]
Super nice. I'm very impressed!

File: react/week5/githubuser-search/src/Components/User/index.js (hunk 4 above)
Comment (daxiousm) [reply react/week5/githubuser-search/src/Components/User/index.js:2]: all the credit goes to Mads, he thought us very well.

File: react/week5/githubuser-search/src/Components/User/User.js (hunk 3 above)
Comment (daxiousm) [reply react/week5/githubuser-search/src/Components/User/User.js:31]: yes it is tachyons css toolkit :)

**Pull request merged**